### DISCUSSION TITLE
Beta 3 vs. v-data-table

### DISCUSSION THREAD

**Discussion by AquaMCU** [vuetifyjs/vuetify#14988] [General]
Hello Team. 
I have installed the beta 3 and try to write a small app using the v-data-table directive. Unfortunately, it says: "Failed to resolve component: v-data-table". Then I look through the documentation of NEXT, I see, that is also not included in the docs? 

Will the data table component not be included in vuetify 3?

Oliver

**Comment by alex-deneuvillers** [comment]
Hello, there is a branch in WIP [here](https://github.com/vuetifyjs/vuetify/tree/feat/v3-data-table), I can't imagine vuetify without data table :)

**Reply by mjzarrin** [reply]
Could you please explain, how to install it?
I tried this command but it didn't work.

`yarn add "https://github.com/vuetifyjs/vuetify.git#feat/v3-data-table" `

**Reply by ghost** [reply]
I would also really like to know this :)

**Reply by ardeay** [reply]
Nice, I'm excited to try when this is ready!

**Reply by HengYuLuck** [reply]
I have the same problem :)

**Comment by nekosaur** [comment]
The v-data-table component is not ready for use. The branch mentioned is a work in progress.

**Reply by mrrrk** [reply]
Hmmm.  On that basis, perhaps this should be an **alpha** version rather than a beta?  This has stopped me in my tracks somewhat and I wouldn't have started converting from 2 -> 3 if I'd known there were bits missing!  As far as I understand, the definition of a beta version is that it is complete but with potential bugs.  The library is not complete! :-)

**Reply by KaelWD** [reply]
It's as complete as 3.0 is going to be, there are other milestones for more features: https://github.com/vuetifyjs/vuetify/milestone/56, https://github.com/vuetifyjs/vuetify/milestone/53

**Reply by mrrrk** [reply]
My apologies.  I've misunderstood what Beta 3.0 comprised.  I'll hang on until 3.1 or 3.2.

**Reply by 1Luc1** [reply]
the corresponding task for v-data-table https://github.com/vuetifyjs/vuetify/issues/13479

**Comment by RakeshAMD** [comment]
Is this issue resolved?

**Comment by adnan-razzaq** [comment]
Any update on beta release of v-data-table?

**Comment by MaYoRr** [comment]
Where is v-data-table?

**Comment by ryuchaehwa** [comment]
any updates?

**Comment by 1Luc1** [comment]
v-data-table is currently available within [Vuetify Labs](https://dev.vuetifyjs.com/en/labs/introduction/#vuetify-labs), which is a new way for developers to use unfinished components in an alpha state. Labs is available since version [3.1.0](https://github.com/vuetifyjs/vuetify/releases/tag/v3.1.0). See [this description](https://dev.vuetifyjs.com/en/labs/introduction/#usage) for how to use the data-table component.

**Reply by mferris77** [reply]
Thanks for the updated info - however the description link goes to a 404 page.

**Reply by 1Luc1** [reply]
you are welcome. just checked the links; do work on my side

**Comment by codeforcrack-kalai** [comment]
3.1.11 its working  perfectly

**Comment by kalai-selva** [comment]
vutify 3.1.x data table not working with  "nuxt": "^3.2.3",

**Comment by tit1** [comment]
Any word on when data-table will be stable? i use it almost exclusively in my app and can up upgrade to vuetify 3 until its ready.

**Comment by teckel12** [comment]
@tit1 While you can use v-data-table by importing Vuetify Labs, v-data-table is kinda useless with all the missing features. For example, custom per column filtering is not present.  So, you can't do something like have filters for date, company, first, last name and have those filters work per column.  It only supports a single filter for the entire form.  This is simply not how my clients want to have filters.  Also, many filters have drop-downs specific to a column.  Or sometimes there's a true/false value which needs to be conditional based on the specific column.  For example, signed true/false and synced true/false.  Signed and synced are two different filters which need to filter for specific columns.  Vuetify 3 doesn't support this at all anymore, making the data-table basically useless for every project I have.

I should have stuck with Vue 2, Vue 3/Vuetify 3 is a hot-mess.

**Comment by mattStayner** [comment]
Any update on when data-table will be stable? I has 3 large apps that I want to migrate to Vue3, but I'm hesitant to do so until the data-table issue is out of alpha.

**Comment by Stefanosapk** [comment]
Any news on when the data-table component will be stable?

**Comment by Pes8** [comment]
is vuetify maintained? years from v3 and no data table, no calendar

**Reply by jacekkarczmarczyk** [reply]
![image](https://github.com/vuetifyjs/vuetify/assets/15625235/8107f55f-e87d-499d-a130-a4956878cca7)

@Pes8: Looks maintained for me. Calendar is on the way, but data table is there already
